docs/pages/enterprise/getting-started.mdx

```diff
@@ -443,7 +434,7 @@ This example command will create a Teleport user called `testuser` which has the
 `access` role. Feel free to change these to suit your needs.
 
 ```code
-$ docker exec teleport tctl users add testuser --roles=access
+$ docker exec teleport tctl users add testuser --roles=access --logins=root,ec2-user=ubuntu
```

Comment:
Just double-checking that final `=` should be there—I'm not sure I've seen that syntax in the value of `logins` before.

Reply:
yes, thanks.  corrected and made in the format of the earlier tctl users add